test/RachisTests/RavenDB-6602.cs

```diff
@@ -48,7 +48,10 @@ private class User
 
                 using (var session = store.OpenSession(databaseName))
                 {
-                    Assert.Throws<AllTopologyNodesDownException>(() => session.Load<User>("users/1"));
+                    var e = Assert.Throws<RavenException>(() => session.Load<User>("users/1"));
+                    Assert.Contains("System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> " +
+                                    "System.Net.Http.WinHttpException: A connection with the server could not be established"
```

Comment:
This is not a good idea. This will fail on Linux, for example.
I think we can just validate the `RavenException` here, and not bother with the message